SetupDataPkg/Library/ConfigDataLib/ConfigDataLib.c

```diff
@@ -92,7 +92,6 @@ IterateConfData (
                    (CdataHdr->Length << 2) - sizeof (*CdataHdr) - sizeof (CDATA_COND) * CdataHdr->ConditionNum
                    );
         if (EFI_ERROR (Status)) {
-          ASSERT_EFI_ERROR (Status);

```

Comment:
Remove this assert to prevent crashing the system when write protected config data is passed in to a debug build system.